File: java/two/Two.java
import java.util.ArrayList;
import java.util.Random;

public class Two {
	public static void main(String[] args) {
		ArrayList<Shape> shapes = new ArrayList<Shape>();
		Random rand = new Random();
		for(int x = 0; x < 10; x++) {
			double foo = x * rand.nextDouble() + 1;
			double bar = x * rand.nextDouble() + 1;
			shapes.add(new Circle(foo));
			shapes.add(new Rectangle(foo, bar));
			shapes.add(new Square(bar));
		}
		
		for(Shape shape : shapes) {
			System.out.println(shape.getArea());
			System.out.println(shape.getInfo());
			System.out.println();
		}
	}
}

File: python/two/two.py
import random
from rectangle import Rectangle
from square import Square

shapes = []

for x in range(20):
    rect_x = x * random.randint(1,10) + 1
    rect_y = x * random.randint(1,10) + 1
    if random.randint(1,10) > 5:
        shapes.append(Rectangle(rect_x, rect_y))
    else:
        shapes.append(Square(rect_x))

for shape in shapes:
    print(shape.get_area())
    print(shape.get_info())
    print
